db/schema.rb

```diff
@@ -1087,6 +1087,16 @@
     t.index ["user_id"], name: "index_user_counters_on_user_id", unique: true
   end
 
+  create_table "user_optional_fields", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.string "label"
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
```

Comment:
Not a huge deal but should this also have `null: false`? Or would that be unnecessary because of the foreign key?

Reply:
Looking at a bunch of the other tables it doesn't look like any of them have `null: false` on their foreign keys.

Reply:
The decision has this rationale: do we want user optional fields in the DB that don't belong to a user? In this case I'd say no, so as Andy suggested this should have `null: false`. 

Admittedly there are cases in the DB where their FK columns should have `null: false`, we'll eventually fix that, but as this is a new table, we can get it right the first time :D

BTW you should add it (and the other ones) to the migration and re-run it, as you can't have a schema misaligned with its migration.

Reply:
Got it!

Added it to `user_id` and to the migrations as well.